charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -41,6 +41,15 @@ spec:
   {{- if .Values.volumes }}
   volumes:
 {{ toYaml .Values.volumes | indent 4 }}
+  {{- end }}
+  volumePermissions: {{ .Values.volumePermissions.enabled }}
+  {{- if .Values.volumePermissions.enabled }}
+  initContainers:
+  - name: volume-perms-init
```

Comment:
the name field is no longer needed since it's being set to a fixed value in the code.

Reply:
we cant exclude name field  in order to comply with openAPi schema validation. Container name is a mandatory field